pkg/mobile/serviceInstances.go

```diff
@@ -17,5 +18,37 @@ func NewServiceInstanceLister(scClient scv1beta1.ServicecatalogV1beta1Interface)
 
 func (lister *ServiceInstanceListerImpl) List(namespace string) (*ServiceInstanceList, error) {
 	listOpts := v1.ListOptions{}
-	return lister.scClient.ServiceInstances(namespace).List(listOpts)
+	getOpts := v1.GetOptions{}
+	serviceInstanceList := ServiceInstanceList{}
+
+	serviceInstances, err := lister.scClient.ServiceInstances(namespace).List(listOpts)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, si := range serviceInstances.Items {
+		sc, err := lister.scClient.ClusterServiceClasses().Get(si.Spec.ClusterServiceClassRef.Name, getOpts)
+		if err != nil {
+			return nil, err
+		}
+
+		if isMobileService(sc) {
+			serviceInstanceList.Items = append(serviceInstanceList.Items, si)
+		}
+	}
+
+	return &serviceInstanceList, nil
+}
+
+func isMobileService(sc *v1beta1.ClusterServiceClass) bool {
+	return contains(sc.Spec.Tags, "mobile-service")
+}
+
+func contains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
```

Comment:
Should not better here adopted names instead of a letter for the vars `a`, and `s`, and `e` in order to clarify its propose?  Something as which follows. 

```go 
func contains(arr []string, value string) bool {
      for _, arrValue := range arr {
		if arrValue == value {
			return true
		}

```

Also,

Reply:
Also, should not better keep this kind of implementation in utils file ?

Reply:
for a long program, yes. But for a short program like this, unless the code is very very hard to understand, then it is ok to use short var names. It is very common in Golang where short var names are used.

Reply:
I'm not against a utils file but I tend to move functions into one as the need arises.

Reply:
@wei-lee I found this [here](https://github.com/golang/go/wiki/CodeReviewComments#variable-names), _Variable names in Go should be short rather than long_. It is really interesting.  👍

Reply:
@camilamacedo86  :+1:

Reply:
@camilamacedo86 thanks for sharing. Good read indeed.